nntrainer/layers/fc_layer.cpp

```diff
@@ -43,25 +43,26 @@ int FullyConnectedLayer::initialize(Manager &manager) {
     throw std::invalid_argument("Fully connected layer takes only one input");
   }
 
-  output_dim[0] = input_dim[0];
-  output_dim[0].width(unit);
+  auto &in_dim = input_dim[0];
+  in_dim.setEffDimFlag(0b1001);
```

Comment:
It is not always true for rnn and lstm. Effective Dimension could be 1011.

Reply:
Yes, I'll add a todo. There are some subtle problem regarding this I'd like to discuss later.